src/OpenLoco/src/Audio/Audio.cpp

```diff
@@ -1027,68 +1027,89 @@ namespace OpenLoco::Audio
         }
     }
 
-    static void addAllSongsToPlaylist(int32_t excludeTrack, std::vector<uint8_t>& playlist)
+    static void addAllSongsToPlaylist(std::vector<uint8_t>& playlist)
     {
         for (auto i = 0; i < kNumMusicTracks; i++)
         {
-            if (i != excludeTrack)
-            {
-                playlist.push_back(i);
-            }
+            playlist.push_back(i);
         }
     }
 
-    static void addCurrentEraSongsToPlaylist(int32_t excludeTrack, std::vector<uint8_t>& playlist)
+    static void addCurrentEraSongsToPlaylist(std::vector<uint8_t>& playlist)
     {
         auto currentYear = getCurrentYear();
         for (auto i = 0; i < kNumMusicTracks; i++)
         {
             const auto& mi = kMusicInfo[i];
             if (currentYear >= mi.startYear && currentYear <= mi.endYear)
             {
-                if (i != excludeTrack)
-                {
-                    playlist.push_back(i);
-                }
+                playlist.push_back(i);
             }
         }
     }
 
-    static int32_t chooseNextMusicTrack(int32_t excludeTrack)
+    static void addCustomSelectionSongsToPlaylist(std::vector<uint8_t>& playlist)
```

Comment:
we can make all of these 'adds' just return the vector instead of taking a mutable vector